README.md

```diff
@@ -29,23 +29,145 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
 
 ## Motivation
 
-* Users need all types of duration format depending on the requirements of their application. For example, in the airlines websites, to show how long a flight takes, the duration should be in Short or Narrow format
+* Users need all types of duration format depending on the requirements of their application. For example, to show how long a flight takes, the duration should be in Short or Narrow format
   - 1 hr 40 min 60 sec → Short 
   - 1 h 40 m 60 sec  → Narrow
 
-* Furthermore, for TTS (text to speech) applications, Wide format is needed
-  - 1 hour, 40 minutes and 60 seconds.
-
+## Requirements & Design
+
+In this section, we are going to illustrate each user needs (requirements) and a design for each need (requirement)
+
+### Input Value
+  * Users needs to identify the input values
+  * For example
+    * format( 100 seconds)
+    * format( 1 hour, 20 minutes)
+
+#### Design
+  * Input value will be an object from `Temporal.Duration`
+  * Example:
+     * `const d = new DurationFormat();`
+     * `d.format(new Temporal.Duration({hour: 3, minute: 4});`
+
+
+### Formatting width
+Users want to determine several types of the formatting width as following
+  | Format width  | Example               |
+  | -----------   | -----------           |
+  |  Wide         |1 hour and 50 minutes  |
+  |  Short        |1 hr, 50 min           |
+  |  Narrow       |1h 50 m                |
+  |  Numeric      |                       |
+  |  Dotted       |1: 50: 00              |
+
+
+#### Design
+  * The user can determine the formatting width using a  parameter `style` and the value of this parameter take a ` string` as following:
+      * “wide” 
```

Comment:
Please make the names string literals: \`"wide"\` (rendered as `"wide"`)

Reply:
I propose that we use `"long"` instead for consistency with ECMA 402 (`DateTimeFormat`, `RelativeTimeFormat`, `ListFormat`) since I'm not even sure if consistency with ICU is necessary.

Reply:
+1 on long/short/narrow instead of wide/short/narrow

Reply:
Done !